addons/hr_recruitment_survey/models/survey_survey.py

```diff
@@ -0,0 +1,23 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, fields, models
+
+class SurveySurvey(models.Model):
+    _inherit = 'survey.survey'
+
+    survey_type = fields.Selection(selection_add=[('recruitment', 'Recruitment')], ondelete={'recruitment': 'set default'})
+    job_ids = fields.One2many("hr.job", "survey_id")
+
+    @api.depends('survey_type')
+    @api.depends_context('uid')
+    def _compute_allowed_survey_types(self):
+        super()._compute_allowed_survey_types()
+        if self.env.user.has_group('hr_recruitment.group_hr_recruitment_interviewer') or \
+           self.env.user.has_group('survey.group_survey_user'):
+            self.allowed_survey_types = (self.allowed_survey_types or []) + [('recruitment', 'Recruitment')]
+
+    def get_formview_id(self, access_uid=None):
+        if self.survey_type == 'recruitment' and not self.env.user.has_group('survey.group_survey_user'):
+            if view_id := self.env.ref('hr_recruitment_survey.survey_survey_view_form'):
```

Comment:
As it is a record, no need of "_id" suffix
```suggestion
            if view := self.env.ref('hr_recruitment_survey.survey_survey_view_form'):
```

Reply:
Also, please add
```suggestion
            if view := self.env.ref('hr_recruitment_survey.survey_survey_view_form', raise_if_not_found=False):
```